scripts/merge-and-build.sh

```diff
@@ -114,6 +139,8 @@ else
   fi
 fi # End check if we are master
 
+# TODO: Origin already vendors the web console. Should we stop updating it separately for OSE?
```

Comment:
It doesn't seem to change anything. We can still soft-reset everything, recreate the web client branding commit, recreate the specfile commit, maybe add a new bump for the webconsole, tito tag.